src/treeview/nodes/targets.ts

```diff
@@ -103,7 +103,8 @@ export class TargetNode extends BaseNode {
   async getTreeItem() {
     const item = super.getTreeItem() as vscode.TreeItem;
 
-    item.label = `${this.target.name} ${this.target.type}`;
+    item.label = this.target.name;
+    item.tooltip = `${this.target.name} (${this.target.type})`;
```

Comment:
Is it worth repeating the target name in the tooltip?

Reply:
I don't think so.